common/app/components/SubtitlePlayer.tsx

```diff
@@ -239,6 +259,26 @@ const SubtitleRow = React.memo(function SubtitleRow({
     const disabledClassName = disabled ? classes.disabledSubtitle : '';
     const { t } = useTranslation();
 
+    const processedText = useMemo(() => {
+        let text = subtitle.text;
+
+        if (subtitleHtml === SubtitleHtml.remove && convertNetflixRuby) {
```

Comment:
I think for this feature you can just totally ignore the `subtitleHtml` setting. This setting can operate independently of that.